superduperdb/vector_search/atlas.py

```diff
@@ -198,7 +198,11 @@ def _create_index(self, collection: str, output_path: str):
         self.database.command(index_definition)
 
     def _check_if_exists(self, index: str):
-        indexes = self.index.list_search_indexes()
-        return len(
-            [i for i in indexes if i['name'] == index and i['status'] == 'READY']
-        )
+        try:
+            indexes = self.index.list_search_indexes()
+            return len(
+                [i for i in indexes if i['name'] == index and i['status'] == 'READY']
+            )
+        except pymongo.errors.OperationFailure as e:
```

Comment:
Why ignore the exception?

Reply:
I am returning False when error occurs, then it will create_index, as `listSearchIndexes` is deprecated. temporary solution.

```
if not self._check_if_exists(identifier):
            self._create_index(collection, output_path)
```

Reply:
Isn't there an alternative to `listSearchIndexes`?

Reply:
@blythed there is no alternative for now

Reply:
Does the index successfully create and also get deleted?